parser/rdjson.go

```diff
@@ -0,0 +1,40 @@
+package parser
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+
+	"github.com/reviewdog/reviewdog/proto/rdf"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+var _ Parser = &RDJSONParser{}
+
+// RDJSONParser is parser for rdjsonl format.
+type RDJSONParser struct{}
+
+func NewRDJSONParser() *RDJSONParser {
+	return &RDJSONParser{}
+}
+
+func (p *RDJSONParser) Parse(r io.Reader) ([]*rdf.Diagnostic, error) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported method RDJSONParser.Parse should have comment or be unexported